Assets/Scripts/ItemSystem/ItemInfo.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Cosmobot.ItemSystem
+{
+    [CreateAssetMenu(fileName = "ItemInfo", menuName = "Cosmobot/ItemSystem/ItemInfo", order = 1)]
+    public class ItemInfo : ScriptableObject, IEquatable<ItemInfo>
+    {
+
+        [SerializeField]
+        private string id;
+        [SerializeField]
+        private string displayName;
+        [SerializeField]
+        private Texture2D icon;
+        [SerializeField]
+        private GameObject prefab;
+        [SerializeField]
+        private SerializableDictionary<string, string> additionalData = new();
+
+        public string Id => id;
+        public string DisplayName => displayName;
+        public Texture2D Icon => icon;
```

Comment:
It's unused, but I guess it exists for future purposes?

Reply:
yep